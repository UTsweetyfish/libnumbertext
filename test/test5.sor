# check boundary detection in nested calls
__numbertext__
^1$ uno
^1 un
(1)000000 $($($1)) million
(1)(000001) $($($1)) million $(|$($2))
(\D+) \1
