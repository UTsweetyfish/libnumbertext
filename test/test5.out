un million
un million uno
